lib/tasks/scripts/covid_medication_reminders.rb

```diff
@@ -0,0 +1,10 @@
+class CovidMedicationReminders
+  def self.call(number_of_patients: 10_000)
+    if CountryConfig.current[:name] != "India" || !SimpleServer.env.production? || !Flipper.enabled?(:experiment)
+      raise "Cannot send Covid medication reminders in this env"
```

Comment:
This will give a confusing error message if it hits the last branch for the feature flag being disabled...may be worth having its own message for that.